### PR TITLE
Ambundo homepage section

### PR DESCRIPTION
**Pull request by Katiechi**

### CODE REVIEW

File: client/src/components/PortfolioModal.js [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+        Continue Reading
+      </Button>
+
+      <Modal
+        show={show}
+        onHide={handleClose}
+        backdrop="static"
+        keyboard={false}
+      >
+        <Modal.Header closeButton>
```
Comment (munenebrian) [inline client/src/components/PortfolioModal.js:24]: make sure to research more on the line of code.

**Review by munenebrian — APPROVED** [review]

File: client/package.json [hunk 2]
```diff
@@ -3,17 +3,24 @@
...
     "react": "^18.2.0",
+    "react-bootstrap": "^2.5.0",
     "react-dom": "^18.2.0",
     "react-router-dom": "^6.4.2",
     "react-scripts": "5.0.1",
     "web-vitals": "^2.1.4"
   },
   "scripts": {
-    "start": "react-scripts start",
+    "start": "PORT=3006 react-scripts start",
```
Comment (chibukasia) [inline client/package.json]: Can we use port 4000 instead of 3006?

**Review by chibukasia — CHANGES_REQUESTED** [review]
Add descriptive comments on what the code is doing

File: client/package.json (hunk 2 above)
Comment (Katiechi) [reply client/package.json]: Yes sure, ill change it

**Review by chibukasia — APPROVED** [review]

**Pull request merged**